readthedocs/core/static-src/core/js/doc-embed/search.js

```diff
@@ -32,6 +32,7 @@ function attach_elastic_search_query(data) {
                 var total_count = data.count || 0;
 
                 if (hit_list.length) {
+                    console.debug('Read the Docs search got a result. Showing results.')
```

Comment:
We should drop debug/log statements like this for prod. You can tell if the Sphinx indexes are used as the search result return will be empty -- or easier, you'll see a flood of requests for Sphinx's index files.

I do think this is helpfu though. I think an addition to our JS could be to log when DEBUG = True, but I took a really quick swing at this and hit issues. We need to pass through our footer most likely. I'd say lets remove these statements and find a method of exposing DEBUG to our JS.